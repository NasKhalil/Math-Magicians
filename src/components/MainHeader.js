import React from 'react';
import { NavLink } from 'react-router-dom';
import classes from './MainHeader.module.css';

export default function MainHeader() {
  return (
    <div>
      <header className={classes.header}>
        <nav>
          <h2>Math magicians</h2>
          <ul>
            <li>
              <NavLink activeClassName={classes.active} to="/">
                Home
              </NavLink>
            </li>
            <li>
              |
            </li>
            <li>
              <NavLink activeClassName={classes.active} to="/calculator">
                Calculator
              </NavLink>
            </li>
            <li>
              |
            </li>
            <li>
              <NavLink activeClassName={classes.active} to="/quotes">
                Quotes
              </NavLink>
            </li>
          </ul>
        </nav>
      </header>
    </div>
  );
}
