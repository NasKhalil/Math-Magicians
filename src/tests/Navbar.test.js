import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import MainHeader from '../components/MainHeader';

it('renders correctly', () => {
  const nav = renderer
    .create(
      <BrowserRouter>
        <MainHeader />
      </BrowserRouter>,
    )
    .toJSON();
  expect(nav).toMatchSnapshot();
});
