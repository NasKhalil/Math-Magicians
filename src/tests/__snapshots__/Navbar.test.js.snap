// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className="header"
  >
    <nav>
      <h2>
        Math magicians
      </h2>
      <ul>
        <li>
          <a
            activeClassName="active"
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            activeClassName="active"
            className=""
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            activeClassName="active"
            className=""
            href="/quotes"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
